app/components/submit-data.js

```diff
@@ -42,6 +45,17 @@ const dataMap = {
     caseId: 'ccdCase.id',
     caseState: 'ccdCase.state',
     registry: 'registry',
+    caseType: 'caseType',
+    deceasedHasAssetsOutsideUK: 'assets.assetsoverseas',
+    foreignAsset: 'assets.assetsoverseas',
```

Comment:
I'll pretend I haven't seen this 🙈

Reply:
These are not the assets your looking for!! So says Femi anyway..